src/v2/Apps/Artwork/Components/ArtworkBanner/ArtworkBanner.tsx

```diff
@@ -0,0 +1,189 @@
+import { ArtworkBanner_artwork } from "v2/__generated__/ArtworkBanner_artwork.graphql"
+import { ArtworkBannerQuery } from "v2/__generated__/ArtworkBannerQuery.graphql"
+import { SystemContext } from "v2/System"
+import { renderWithLoadProgress } from "v2/System/Relay/renderWithLoadProgress"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import React, { useContext } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import {
+  Box,
+  ChevronIcon,
+  Column,
+  Flex,
+  GridColumns,
+  Image,
+  Text,
+} from "@artsy/palette"
+import { TopContextBar } from "v2/Components/TopContextBar"
+import { RouterLink } from "v2/System/Router/RouterLink"
+
+export interface ArtworkBannerProps {
+  artwork: ArtworkBanner_artwork
+}
+
+export const ArtworkBanner: React.FC<ArtworkBannerProps> = props => {
+  const bannerProps = computeBannerProps(props)
+
+  if (!bannerProps) {
+    return null
+  }
+
+  const { image, meta, name, subHeadline, href } = bannerProps
+  const isShow = props.artwork?.context?.__typename === "Show"
+
+  return (
+    <RouterLink to={href} textDecoration="none">
```

Comment:
I'm not sure if it matters but I think this anchor tag is collapsed without `display: block;`?

Reply:
Nah its all good